### DISCUSSION TITLE
How to print a map in scale on A4 paper 1:2000 (1 cm : 2000 meters)

### DISCUSSION THREAD

**Discussion by codebombi** [fleaflet/flutter_map#1975] [Q&A] [answered]
Please tell us how to print a map in scale on A4 paper 1:2000 (1cm:2000meters)? Thank you in advance!

**Comment by JaffaKetchup** [comment]
Hey, https://wiki.openstreetmap.org/wiki/OSM_on_Paper is probably the best place to start :)

Otherwise your question is a little bit out of scope. Can you provide any more details?

**Reply by codebombi** [reply]
Thanks for the link, it's about services for OSM. I want to make a package for flutter_map, printing on different A4 A3 A2 formats... but I don't understand how I can calculate such a scale and take a screenshot. Please tell me which way I should look to create such a package.

**Reply by JaffaKetchup** [reply]
Unfortunately then this is quite out of scope for us to answer. You can try another 3rd party package like screenshot, then do something with that? Please note that you'll need to stick to all the rules implemented by tile servers as well.

**Comment by codebombi** [comment, marked as answer]
Please tell me how to implement for fit Camera - Camera Fit relative to Marker? (Because Marker has only one point and there is no way to build LatLngBounds) or How do I set the zoom relative to the scale in meters?
<img width="191" alt="Screenshot 2024-10-15 at 16 03 34" src="https://github.com/user-attachments/assets/4d33a11c-0bd2-4d40-8f66-1b097e1c7eeb">

**Reply by codebombi** [reply]
I built a simple polygon and LatLngBounds)
<img width="450" alt="Screenshot 2024-10-15 at 22 05 07" src="https://github.com/user-attachments/assets/2726c74d-5cbf-4507-95a0-dd2a169742d7">
